lib/Signature.js

```diff
@@ -43,13 +43,13 @@ class Signature {
     /*
         Signature is a HMAC-SHA256 encoded message containing nonce, customer ID and API key.
         The HMAC-SHA256 code must be generated using a secret key that was generated with your
-        API key. This code must be converted to it's hexadecimal representation 
+        API key. This code must be converted to it's hexadecimal representation
         (64 uppercase characters).
     */
     _createSignature(){
         const nonce = this._generateNonce();
         const message = nonce + this.clientId + this.key;
-        const signer = crypto.createHmac("sha256", new Buffer(this.secret, "utf8"));
+        const signer = crypto.createHmac("sha256", encodeURI(this.secret, "utf8"));
```

Comment:
please use `Buffer.from(this.secret, "utf8")` instead of `encodeURI `

Reply:
@krystianity I used `encodeURI` because `Buffer.from` is deprecated

![Screenshot 2021-02-18 at 10 41 00 AM](https://user-images.githubusercontent.com/10850562/108337790-cc084980-71d5-11eb-9b63-d582437a4d2f.png)